### DISCUSSION TITLE
Rofi Theme Selector won't run

### DISCUSSION THREAD

**Discussion by Rpergy** [davatorium/rofi#1798] [General]
### Rofi version (rofi -v)

Version: 1.7.5

### Configuration

https://gist.github.com/Rpergy/496daa6bfa287d47d1ce6cc210ae1129

### Theme

https://gist.github.com/Rpergy/e49d020909dea1598cf5910f167e2a49

### Launch command

rofi-theme-selector

### Step to reproduce

* Open terminal (in order to see the output)
* Run `rofi-theme-selector`

### Expected behavior

Opening the Rofi Theme Selector application

### Actual behavior

The program returns this error message whenever I run `rofi-theme-selector`:
`/usr/bin/rofi-theme-selector: line 6: which: command not found
/usr/bin/rofi-theme-selector: line 7: which: command not found
/usr/bin/rofi-theme-selector: line 8: which: command not found
Did not find 'sed', script cannot continue.`

### Using wayland display server protocol

- [X] No, I don't use the wayland display server protocol

**Comment by DaveDavenport** [comment]
(sorry read wrong, it goes wrong before sed)
On what distro are you running this one?  it reports that 'which' is not found. this is normally available?

Moving to discussion, as this seems to be an issue with dependencies, not a bug.

**Comment by skunkdog** [comment]
Have similar problem using openbsd. Normaly "which sed" finds /bin/sed but rofi can't find it. 
$ rofi -show drun
Did not find 'sed', script cannot continue.

**Reply by DaveDavenport** [reply]
```
$ rofi -show drun
```

Does not normally call sed, what extra scripts do you have configured for rofi to load?
Maybe pastebin the output of `rofi -info`?

**Reply by skunkdog** [reply]
Im sorry, i meant that running rofi -show drun and choosing rofi-theme-selector show this error "Did not find 'sed', script cannot continue. Same error shows if i run "rofi-theme-selector" from command line.

**Reply by DaveDavenport** [reply]
What version ?  Please provide some info.

**Reply by skunkdog** [reply]
rofi version: 1.7.9.1
openbsd version: openbsd 7.9 amd64 ~current 
No rofi configuration 
Using X11

**Reply by DaveDavenport** [reply]
You are on a old version.

New version uses `command ` to find the executable path.
https://github.com/davatorium/rofi/blob/next/script/rofi-theme-selector
